dashboard/src/pages/TreeDetails/Tabs/CommitNavigationGraph.tsx

```diff
@@ -153,12 +154,21 @@ const CommitNavigationGraph = (): JSX.Element => {
       min: 100,
       data: xAxisIndexes,
       valueFormatter: (value: number, context): string => {
+        const currentCommitData = commitData[value];
+        const currentCommitDateTime = formatDate(
+          z.string().catch('').parse(currentCommitData.earliestStartTime),
```

Comment:
`earliestStartTime` is already a string right? why do we need to parse it to string here?